get_bams/1.0/get_bams.R

```diff
@@ -0,0 +1,510 @@
+#!/usr/bin/env Rscript
+#
+# Usage:
+#   get_bams.R <input_table> <output_table> [<library_id_column>]
+#
+# Notes:
+#   - <library_id_column> can be a name or the number 1. If it is the
+#     number 1, the script assumes that the input file has no header
+#     and only has a single column. The default value is 'library_id'
+#     (i.e., the script assumes a file header).
+#
+#   - The column names are prefixed with 'al.', 'lc.', and 'lb.' for
+#     designating alignment-, libcore-, and library-specific columns.
+#     It also helps with ensuring no collisions when joining the
+#     obtained information with the input table.
+#
+
+
+# Load packages -----------------------------------------------------------
+
+suppressWarnings(suppressPackageStartupMessages({
+
+message("Loading packages...")
+
+required_packages <- c(
+  # Included to avoid mysterious error
+  "xml2",
+  # Password prompt
+  "getPass",
+  # Data handling
+  "readr",
+  "dplyr",
+  "purrr",
+  "tidyr",
+  "lubridate",
+  # Web requests
+  "httr",
+  "jsonlite"
+)
+
+installed_packages <- rownames(installed.packages())
+
+for (pkg in required_packages){
+  if (!pkg %in% installed_packages) {
+    message(paste0("Installing ", pkg, "..."))
+    install.packages(pkg, repos = "https://mirror.rcg.sfu.ca/mirror/CRAN/")
+  }
+  library(pkg, character.only = TRUE)
+}
```

Comment:
I consolidated the code for installing and loading the libraries so that the libraries aren't listed twice. 

@lkhilton: Are you okay with this approach? I don't think there's a way to prompt the user without installing a package first, which would defeat the purpose of asking in the first place. 

If you're okay with it, I can go ahead and merge this into master.

Reply:
Sure. If we get complaints from folks down the road, we can maybe implement a command line option to by default decline package installation, and users can opt in if they wan it.